group7_ws/src/auefinals/aue_finals/scripts/wall_follower_obstacle_avoidance.py

```diff
@@ -0,0 +1,101 @@
+#! /usr/bin/env python3
+
+import rospy
+from geometry_msgs.msg import Twist
+from sensor_msgs.msg import LaserScan
+import numpy as np
+
+def update_scan(scan_data):
+    global scd
+    scd = scan_data.ranges
+
+def laser_sub():
+    rospy.Subscriber("/scan", LaserScan, callback=update_scan)
+
+def bot_control():
```

Comment:
Let's change this name to something that is related to the function. This will be misleading since everything we do is bot_control